### ISSUE TITLE
changes 

### ISSUE THREAD

**Issue by Achapko200** [hunter-classes/spring-2023-13500-lab-08-mattlabarca10#2]
I would not make so many cpp files because you need an h file for each and then it gets confusing when making the make file

**Comment by mattlabarca10** [comment]
makes sense, I was thinking the same thing the program is quite crowded, thanks!
